mappings/net/minecraft/util/dynamic/Codecs.mapping

```diff
@@ -89,33 +143,56 @@ CLASS net/minecraft/unmapped/C_tyjpezxh net/minecraft/util/dynamic/Codecs
 	METHOD m_ljxipwty validate (Lcom/mojang/serialization/MapCodec;Ljava/util/function/Function;)Lcom/mojang/serialization/MapCodec;
 		ARG 0 codec
 		ARG 1 validator
+	METHOD m_lmmxgycz (Ljava/lang/Object;)Ljava/lang/Object;
+		ARG 0 value
 	METHOD m_lprwixza either (Lcom/mojang/serialization/Codec;Lcom/mojang/serialization/Codec;)Lcom/mojang/serialization/Codec;
 		ARG 0 first
 		ARG 1 second
+	METHOD m_lqfxmpnv (Ljava/util/Optional;)Ljava/util/OptionalLong;
+		ARG 0 optional
 	METHOD m_lrcwcdqs orCompressed (Lcom/mojang/serialization/MapCodec;Lcom/mojang/serialization/MapCodec;)Lcom/mojang/serialization/MapCodec;
 		ARG 0 uncompressed
 		ARG 1 compressed
 	METHOD m_lruqlezz asOptionalLong (Lcom/mojang/serialization/MapCodec;)Lcom/mojang/serialization/MapCodec;
 		ARG 0 codec
+	METHOD m_lscrzsve (Lnet/minecraft/unmapped/C_ncpywfca;)Lnet/minecraft/unmapped/C_tyjpezxh$C_pdblrsjv;
+		ARG 0 id
 	METHOD m_lxagintb (Lcom/mojang/serialization/DynamicOps;Ljava/lang/Object;)Lcom/mojang/serialization/Dynamic;
 		ARG 1 object
 	METHOD m_lybgbxfe createEqualTypeChecker (Ljava/util/function/Function;)Ljava/util/function/Function;
 		ARG 0 typeGetter
 	METHOD m_lytzwisw validate (Lcom/mojang/serialization/Codec;Ljava/util/function/Function;)Lcom/mojang/serialization/Codec;
 		ARG 0 codec
 		ARG 1 validator
+	METHOD m_mcwzjvkz (Ljava/util/List;)Lcom/mojang/serialization/DataResult;
+		ARG 0 list
+	METHOD m_mfpnjepb (Lorg/joml/Vector3f;)Ljava/util/List;
+		ARG 0 vector3f
```

Comment:
This could be `pos`? or just `vector3`

Reply:
I think there was a reason I did this, I'll look

Reply:
mhm, enclosing method uses `pos`